app/static/src/app/plot.ts

```diff
@@ -141,3 +143,6 @@ export function allFitDataToPlotly(allFitData: AllFitData | null, paletteModel:
         }
     }));
 }
+
+const lineStyles = ["dot", "dash", "longdash", "dashdot", "longdashdot"];
+export const lineStyleForParameterSetIndex = (index: number): string => (lineStyles[index % lineStyles.length]);
```

Comment:
If user is eccentric enough to want more than 5 parameter sets, this will wrap around and re-use line styles. User will be able to tell from trace tooltip which parameter set it belongs to, and we'll shortly put in buttons to hide parameter sets. Alternatively, we could put a limit on max number of parameter sets.